client/X11/xf_keyboard.c

```diff
@@ -465,6 +465,26 @@ BOOL xf_keyboard_handle_special_keys(xfContext* xfc, KeySym keysym)
 	{
 		return TRUE;
 	}
+	// Nicholai Mitchko
+	// Add a keyboard binding to allow switching workspaces in FullScreenMode
```

Comment:
And I forgot to mention that we're not using C++ comments in FreeRDP.

Appart from that the patch looks good, any comment from others ?